### PR TITLE
orx-syphon

### PR DESCRIPTION
**Pull request by reinvdwoerd**
Client and server are both working, and I added some examples.
I'm not sure if the Syphon framework will be loaded correctly this way, please correct me if I'm doing it wrong...

### CODE REVIEW

**Comment by reinvdwoerd** [comment]
I also added Worley Noise and orx-parameters for the noise filters in this pull request, I'll try to keep my changes better organised next time...

**Comment by reinvdwoerd** [comment]
Pardon, I see I forgot to include orx-parameters in orx-noise's build.gradle...

File: orx-syphon/src/main/kotlin/SyphonServer.kt [hunk 1]
```diff
@@ -0,0 +1,63 @@
+
```
Comment (edwinRNDR) [inline orx-syphon/src/main/kotlin/SyphonServer.kt]: Package name should be org.openrndr.extra.syphon

File: orx-syphon/src/main/kotlin/jsyphon/JSyphonServerList.kt [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import java.io.File
+import java.util.ArrayList
+import java.util.HashMap
+
+object JSyphonServerList {
+    val list: ArrayList<HashMap<String?, String?>?>?
+        external get
+
+    init {
+        System.load(File("orx-syphon/src/main/kotlin/jsyphon/libJSyphon.jnilib").absolutePath)
```
Comment (edwinRNDR) [inline orx-syphon/src/main/kotlin/jsyphon/JSyphonServerList.kt]: This does not work, please have a look at my fixes in master

File: orx-syphon/src/test/kotlin/ClientExample.kt [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import org.openrndr.application
```
Comment (edwinRNDR) [inline orx-syphon/src/test/kotlin/ClientExample.kt]: src/test is intended for tests in a quite specific format, let's see if it trips up the build server :)

File: orx-syphon/build.gradle [hunk 4]
```diff
@@ -0,0 +1,6 @@
+dependencies {
```
Comment (edwinRNDR) [inline orx-syphon/build.gradle]: I fixed these to be `implementation`, because you don't want these libraries to be exposed in the user's program.

**Comment by edwinRNDR** [comment]
Great! merged/rebased this into master

**Comment by reinvdwoerd** [comment]
Thanks for the feedback!! :)

**Pull request closed without merging**